### PR TITLE
fix for matters.news

### PR DESCRIPTION
**Pull request by linsui**
Fix pop up sign in banner when scrolling to bottom in an article.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3505,6 +3505,15 @@ CSS
 
 ================================
 
+matters.news
+
+CSS
+.jsx-178004874.container {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3511]: `.jsx-178004874.`
JSX seams to be [React JSX](https://reactjs.org/docs/introducing-jsx.html),
and `178004874` seems to be some randomized number.

So I guess a 
```CSS
section[class$="container"][style^="background-image"]
```
Also isn't a INVERT better?

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (linsui) [reply src/config/dynamic-theme-fixes.config:3511]: Thanks! I'll test it later.

Comment (linsui) [reply src/config/dynamic-theme-fixes.config:3511]: INVERT inverts its child elements.

```
INVERT
section[class$="container"][style^="background-image"]
section[class$="container"][style^="background-image"] > h2
section[class$="container"][style^="background-image"] > p
section[class$="container"][style^="background-image"] > div
```

sshot-009

or

```
CSS
section[class$="container"][style^="background-image"] {
    background-image: none !important;
}
```

sshot-008

which is better?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:3511]: Second one

Comment (linsui) [reply src/config/dynamic-theme-fixes.config]: Updated, thanks!

**Pull request merged**